x/gamm/pool-models/stableswap/msgs.go

```diff
@@ -83,6 +83,16 @@ func (msg MsgCreateStableswapPool) InitialLiquidity() sdk.Coins {
 	return msg.InitialPoolLiquidity
 }
 
-func (msg MsgCreateStableswapPool) CreatePool(ctx sdk.Context, poolID uint64) (types.PoolI, error) {
-	return &Pool{}, types.ErrNotImplemented
+func (msg MsgCreateStableswapPool) CreatePool(ctx sdk.Context, poolId uint64) (types.PoolI, error) {
+	// We assume that FuturePoolGovernor is valid.
+	pool := &Pool{
```

Comment:
Just noticed that in balancer we have [`NewBalancerPool`](https://github.com/osmosis-labs/osmosis/blob/031a6b57e949f7d3e8684141e70020cc74c3aa60/x/gamm/pool-models/balancer/balancer_pool.go#L27) that is being called in its [`MsgCreateBalancerPool`](https://github.com/osmosis-labs/osmosis/blob/031a6b57e949f7d3e8684141e70020cc74c3aa60/x/gamm/pool-models/balancer/msgs.go#L98)

Do you think it makes sense to do something similar here?

Reply:
Yeah we should have a constructor that sets fields to default values and takes a poolID as an arg. Good idea.

Reply:
Agreed, we should do a `NewStableswapPool` function

Reply:
Agreed - added in recent commit!